frontend/src/Components/StatsBox.js

```diff
@@ -1,13 +1,21 @@
-import styled from "styled-components/macro";
-
-export default function StatsBox(){
-    return <Wrapper>
+import styled from 'styled-components/macro'
+import ProgressionPercent from './ProgressionPercent'
+import { Typography } from '@material-ui/core'
 
+export default function StatsBox({ percent, text }) {
+  return (
+    <Wrapper>
+      <Typography variant="body1" color="textPrimary">
+        {text}
+      </Typography>
+      <ProgressionPercent percent={percent} />
     </Wrapper>
+  )
 }
 const Wrapper = styled.div`
   background-color: lightskyblue;
+  width: 95%;
   grid-column: span 2;
```

Comment:
This ties the component to a grid container. Try to separate layout and component styling.

Reply:
Fixed 👍